packages/frontend/src/pages/scaling-activity/props/getActivityView.ts

```diff
@@ -1,5 +1,43 @@
-import { ActivityViewProps } from '../view/ActivityView'
+import { Layer2 } from '@l2beat/config'
+import { ApiActivity } from '@l2beat/types'
 
-export function getActivityView(): ActivityViewProps {
-  return {}
+import { getPercentageChange } from '../../../utils/utils'
+import { ActivityViewEntry, ActivityViewProps } from '../view/ActivityView'
+
+export function getActivityView(
+  projects: Layer2[],
+  apiActivity: ApiActivity,
+): ActivityViewProps {
+  return {
+    items: projects
+      .map((x) => getActivityViewEntry(x, apiActivity))
+      .sort((a, b) => +b.tpsDaily - +a.tpsDaily),
+  }
+}
+
+export function getActivityViewEntry(
+  project: Layer2,
+  apiActivity: ApiActivity,
+): ActivityViewEntry {
+  const SECONDS_IN_A_DAY = 24 * 60 * 60
+  const transactionsWeeklyCount =
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+    apiActivity.projects[project.id.toString()]?.data.at(-1)?.[1] ?? 0
```

Comment:
Should be the sum of all the week, this is the daily count

Reply:
thanks, great catch